go.mod

```diff
@@ -60,6 +60,7 @@ require (
 	google.golang.org/grpc v1.33.2
 	google.golang.org/protobuf v1.25.0
 	gopkg.in/ini.v1 v1.62.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
I suppose this is needed, something we use is not available in v3 but only in v2 ?